adapters/handlers/rest/authz/handlers_authz.go

```diff
@@ -213,7 +213,7 @@ func (h *authZHandlers) removePermissions(params authz.RemovePermissionsParams,
 		policies[params.ID] = append(policies[params.ID], *p)
 	}
 
-	if err := h.AuthorizeRoleScopes(principal, authorization.UPDATE, policies[params.ID], params.ID); err != nil {
+	if err := h.authorizeRoleScopes(principal, authorization.UPDATE, policies[params.ID], params.ID); err != nil {
```

Comment:
if we cover `removePermissions` we should probably also add `deleteRole`. Seems counterintuitive to me that you cannot remove a permission from a role, but then can delete that role :)

Reply:
delete won't succeeded unless you have ALL

Reply:
added